Big-Bumper-Bash/Assets/Scripts/CarScripts/FrontLightsControllerScript.cs

```diff
@@ -0,0 +1,56 @@
+using UnityEngine;
+
+public class FrontLightsControllerScript : MonoBehaviour
+{
+    public Light leftLight;
+    public Light rightLight;
+    private bool lightsOn = false;
+    private Material material;
+    private Color init;
+    private Renderer renderer;
+
+    void Start()
+    {
+        SetLights(false);
+
+        SetInitialColor();
+    }
+
+    void Update()
+    {
+        ToggleLightsOnKey();
+    }
+
+    private void ToggleLightsOnKey()
+    {
+        if (Input.GetKeyDown(KeyCode.L))
+        {
+            lightsOn = !lightsOn;
+            SetLights(lightsOn);
```

Comment:
method functionality unclear from name
```suggestion
            SetLightsActiveStatus(lightsOn);
```